thirdparty/nginx/nginx-1.7.1-zimbra/src/core/ngx_zm_lookup.h

```diff
@@ -62,6 +62,7 @@ struct ngx_zm_lookup_work_s {
     ngx_str_t      username;        /* the original username given by user */
     ngx_str_t      auth_id;         /* GSSAPI auth id (principal) */
     ngx_flag_t     isAdmin;         /* whether is admin       */
+    ngx_flag_t     isZx;            /* is /zx/ to the dedicated handlers */
```

Comment:
Does it makes more sense to use ngx_http_zmauth_type in place of 
ngx_flag_t     isAdmin;
ngx_flag_t     isZx;